HomePage.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
+<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
+<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
+<!--[if gt IE 8]>      <html class="no-js"> <!--<![endif]-->
+<html>
+	<head>
+		<meta charset="utf-8">
+		<meta http-equiv="X-UA-Compatible" content="IE=edge">
+		<title>Homepage</title>
+		<meta name="description" content="">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<link rel="stylesheet" href="Style/style.css">
+	</head>
+
+	<body>
+		<div class="main-container">
+		
+			<header class="homepage-header">
+				<span>Welcome</span>
+				<img src="Assets/Icons/menu.svg" alt="Menu Icon" class="menu-icon"/>
+			</header>
+			
+			<section class="homepage-section">
+					<article>
+						<h1>Hey there. I'm Mariane <br> <span>I'm a software developer </span> </h1>
+						<p> I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, 
+							don’t hesitate to contact me.</p>
+					</article>
+					<nav class="social-media-nav">
+							<a href="">
+								<img src="Assets/Icons/twitter.svg" alt="Twitter"/>
+							</a>
+							<a href="">
+								<img src="Assets/Icons/Linkedin-icon.svg" alt="Linked In"/>
+							</a>
+							<a href="">
+								<img src="Assets/Icons/medium.svg" alt="Medium"/>
+							</a>
+							<a href="">
+								<img src="Assets/Icons/github.svg" alt="Github"/>
+							</a>
+							<a href="">
+								<img src="Assets/Icons/angellist.svg" alt="Angel List"/>
+							</a>
+					</nav>
+
```

Comment:
- [ ] Kindly wrap all these icons using `ul` and `li` tags instead of a nav tag due to SEO purpose :+1:

```
<ul>
  <li>
    <a><img /></a>
  </li>
  <li>
    <a><img /></a>
  </li>
<ul>
```